### DISCUSSION TITLE
Update jellyfin

### DISCUSSION THREAD

**Discussion by ashoktvm** [community-scripts/ProxmoxVE#169] [General]
I tried using apt update and upgrade and also just updating inside the console, but it never updated the server. What can I do?

**Comment by MickLesk** [comment]
If you hit "update" or bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/jellyfin.sh)" in console?

**Reply by Tuxdiver** [reply]
apt-get would replace jellyfin-ffmpeg5 with jellyfin-ffmpeg7 - which it is not doing, because it needs a "full-upgrade" to do so and the "/usr/bin/update" command only does  an "apt-get upgrade"

**Reply by jihemsey** [reply]
I had the same problem, the easiest way to resolve it was to send in console :

apt-get --with-new-pkgs upgrade jellyfin jellyfin-server

**Reply by MickLesk** [reply]
okay good 2 know, that can we fix

**Comment by MickLesk** [comment]
Merged into issue.
